### PR TITLE
Reprocess edited messages

### PR DESCRIPTION
**Pull request by jakobhellermann**
Edited messages are handled by
- clearing all bot reactions
- deleting bot responses
- redoing the usual on_message

### CODE REVIEW

File: bot.py [hunk 1]
```diff
@@ -168,15 +168,31 @@ async def on_message(message: discord.Message):
         log.info(f"Sent 1984 DM to {user.display_name}:\n{dm}")
 
 
+@client.event
+async def on_message_edit(old_message: discord.Message, message: discord.Message):
+    if message.channel.id not in main.fast_project_ids: return
+
+    remove_project_log(message)
```
Comment (jakobhellermann) [inline bot.py]: I don't know if removing the message from the project log is a good idea, but without it the bot wouldn't consider looking at the message again.

**Comment by Kataiser** [comment]
Always reprocessing means that legit edits trigger it. I've considered tracking invalid message posts (so valid files but invalid message content) and reprocessing on edit then, so I'd merge that but not this

**Comment by jakobhellermann** [comment]
Okay, makes sense.

>I've considered tracking invalid message posts (so valid files but invalid message content) and reprocessing on edit then, so I'd merge that but not this

Do you think it would be enough to check if the message has a❌reaction from the bot, and if so reprocess it? That would mean that the validation failed previously and nothing was comitted, so rerunning it should be okay unless I'm missing something.

**Comment by Kataiser** [comment]
Yeah that would work. Technically that would mean people can just add their own reaction but they can already add ⏭️ so whatever

**Comment by jakobhellermann** [comment]
I pushed the change. The code also checks that the reaction came from the bot, so that shouldn't be an issue.

**Pull request merged**